### PR TITLE
add a CursorSet command

### PR DESCRIPTION
**Pull request by mikeandmore**
`CursorSet` can set an absolute mouse cursor position. This is helpful for save and restoring mouse cursor positions.

On IRC, we talked about support some auto mouse move features from CWM. With `CursorSet`, you no longer need an external script/process and `CursorMove` to save and restore the cursor.

### CODE REVIEW

File: fvwm/builtins.c [hunk 1]
```diff
@@ -2385,6 +2385,55 @@ void CMD_CursorMove(F_CMD_ARGS)
...
+		fvwm_debug(__func__, "CursorSet needs 2 arguments");
+		return;
+	}
+	int x = val1 * val1_unit / 100;
+	int y = val2 * val2_unit / 100;
+	unsigned int w = 0, h = 0;
+	struct monitor *m = NULL;
+	TAILQ_FOREACH(m, &monitor_q, entry) {
+		w = m->virtual_scr.MyDisplayWidth;
+		h = m->virtual_scr.MyDisplayHeight;
```
Comment (ThomasAdam) [inline fvwm/builtins.c:2403]: This needs correcting, as it's been a long-standing semantic bug.  `m->virtual_scr.MyDisplayWidth` and `m->virtual_scr.MyDisplayHeight` are not per-monitor values, but an aggregate of all displays connected.  Hence, you either want to set `w`/`h` to `monitor_get_all_widths()`/`monitor_get_all_heights()` respectively outside of the loop, or use `m->si->w`/`m->si->h` for per-monitor values.

I'll clean this up separately.

**Review by ThomasAdam — CHANGES_REQUESTED** [review]
Hi @mikeandmore 

I suppose the idea makes sense, but I really don't want a separate command for this.  If it were going to be included in `fvwm3` then I'd want it part of `CursorMove` somehow.  The long-term goal is to reduce commands, not increase them.

You'll also need to add to the man page.

**Comment by mikeandmore** [comment]
let me work on adding this into `CursorMove`. In that case i'll reuse the code left there before.

**Comment by mikeandmore** [comment]
This commit will reuse the `CursorMove` command.

**Comment by mikeandmore** [comment]
I separate the commit this time.

**Comment by ThomasAdam** [comment]
Thanks.  Now we can clean things up a bit further:

* Since commit 6cc71636cfbf839aeca2107a84026da3cea66585 was introduced by 728e21a67b8ed6a4f16556f8766ed95eff67aece, please squash 6cc71636cfbf839aeca2107a84026da3cea66585 into 728e21a67b8ed6a4f16556f8766ed95eff67aece
* We can probably reword the commits a  better.  For example, I would word cf85ff991100c9a1dcfdd2cc2cb7bde7800a0294 as:
```
GetTwoPercentArguments: remove unneeded PeekToken

Remove an unneeded call to PeekToken as this is never used.
```

**Pull request merged**